### ISSUE TITLE
[full:latest] Can't open files in Theia editor

### ISSUE THREAD

**Issue by marcdumais-work** [theia-ide/theia-apps#142]
I can't open files with the latest published `theiaide/theia-full:latest` (`next` version untested). Each time the following exception appears in the browser's console: 

```
Uncaught (in promise) TypeError: Cannot read property 'createScoped' of undefined
    at t.<anonymous> (monaco-editor-provider.js:209)
    at monaco-editor-provider.js:70
    at Object.next (monaco-editor-provider.js:51)
    at monaco-editor-provider.js:45
    at new Promise (<anonymous>)
    at s (monaco-editor-provider.js:41)
    at t.e.doCreateEditor (monaco-editor-provider.js:203)
    at t.<anonymous> (monaco-editor-provider.js:197)
    at monaco-editor-provider.js:70
    at Object.next (monaco-editor-provider.js:51)
```

Possibly not related at all, but we had a [recent problem](https://github.com/theia-ide/theia-apps/issues/126) with similar symptoms, but the exception was different.

**Comment by marcdumais-work** [comment]
A colleague tells me that the `next` version of `theia-full` does not have this bug.

**Comment by marcdumais-work** [comment]
I was able to reproduce when building the `full image's` (latest.package.json) Theia app outside of docker. By trial-and-error, I determined that removing `yang-theia-extension` makes the problem go away.

**Comment by marcdumais-work** [comment]
Closed via #147
